Python/Product/Analysis/LanguageServer/Server.cs

```diff
@@ -116,14 +110,13 @@ private sealed class ReloadModulesQueueItem : IAnalyzable {
             _queue.Dispose();
         }
 
-        private void TraceMessage(IFormattable message) {
+        public void TraceMessage(string message) {
```

Comment:
The reason behind using `IFormattable` is to avoid string allocation when `_traceLogging` is false.